css/src/components/tag.scss

```diff
@@ -200,6 +200,13 @@ $tag-interactive-color-primary-hover: $primary-invert !default;
 
 		.tag-close {
 			border-inline-start-color: $tag-interactive-divider-color-filled;
+
+			&:hover,
```

Comment:
If this is the source of needing a new item added to the theme map, can we get by without it?

Reply:
This is added to ensure that every tag dismiss button on hover has the 115ea3 dark blue text color. When I remove this, the hover color still works except for the filled dismiss button on hover. The text color remains a black as it currently does now in Atlas.

![image](https://github.com/user-attachments/assets/df9904a5-b65c-4eca-a1d9-956261d4d566)

Reply:
Keeping the rule since we've decided to add new items to the theme map.